server2/Arcadia.Assistant/Arcadia.Assistant.Notifications.Contracts/Models/NotificationMessage.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Arcadia.Assistant.Notifications.Contracts.Models
+{
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    [KnownType(typeof(IReadOnlyDictionary<string, string>))]
```

Comment:
why have IReadOnlyDictionary as knowntype? Also, if anyone puts something but a Dictionary in your message, serializer will fail. So maybe just use implementation as in all other places?

Reply:
Removed